Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/MyResourceManager.cls

```diff
@@ -0,0 +1,109 @@
+VERSION 1.0 CLASS
+BEGIN
+  MultiUse = -1  'True
+END
+Attribute VB_Name = "MyResourceManager"
+Attribute VB_GlobalNameSpace = False
+Attribute VB_Creatable = False
+Attribute VB_PredeclaredId = False
+Attribute VB_Exposed = False
+Option Explicit
+
+Private Const ModuleName As String = "MyResourceManager."
+
+Private Type TRibbonStrings
+    Strings As Rubberduck_RibbonDispatcher.ControlStrings
+End Type
+Private this As TRibbonStrings
+
+Implements Rubberduck_RibbonDispatcher.IResourceManager
+
+Private Function IResourceManager_GetCurrentUIString(ByVal Name As String) As String
+    On Error GoTo EH
+    IResourceManager_GetCurrentUIString = this.Strings.Item(Name)
+
+XT: Exit Function
+EH: ReraiseError Err, ModuleName & "IResourceManager_GetCurrentUIString"
+    Resume XT
+End Function
+
+Private Function IResourceManager_LoadImage(ByVal ImageId As String)
+    On Error GoTo EH
+    IResourceManager_LoadImage = Nothing
+XT: Exit Function
+EH: ReraiseError Err, ModuleName & "IResourceManager_LoadImage"
+    Resume XT
+End Function
+
+Private Sub Class_Initialize()
+    On Error GoTo EH
+    Set this.Strings = New Rubberduck_RibbonDispatcher.ControlStrings
+    With this.Strings
+        .AddControl "BrandingGroup", "RubberDuck", _
```

Comment:
Speaking of branding... it's `Rubberduck`, not `RubberDuck`. 😉

Reply:
I mostly get that correct - this one slipped through the cracks.